src/domain/PowerHistory.cpp

```diff
@@ -93,7 +96,13 @@ void PowerHistory::setContext(TicProcessingContext* context) {
 
 
 void PowerHistory::onNewPowerData(const TicEvaluatedPower& power, const TIC::Horodate& horodate, unsigned int frameSequenceNb) {
-    if (!power.isValid || !horodate.isValid) {
+#ifdef EMBEDDED_DEBUG_CONSOLE
+    Stm32DebugOutput::get().send("onNewPowerData()\n");
+#endif
+    if (!power.isValid /*|| !horodate.isValid*/) {
```

Comment:
This is incompatible with standard TIC.
We should instead use inter-frame time measurement when TIC is historical (see ticket #4)